src/components/help/style/help.less

```diff
@@ -36,3 +36,16 @@
   transition-property: top,left,width,height;
   transition-delay: .01s; /* Start animation when start position is set */
 }
+
+.cadastre.de #catalogHeading button.ga-icon,
+.cadastre.fr #catalogHeading button.ga-icon {
+  display: none;
+}
+
+.cadastre.de #catalogHeading[aria-expanded="true"] span {
+  letter-spacing: -0.4px;
+}
+
+.cadastre.de a#catalogHeading {
+  height: 42px;
+}
```

Comment:
All these css must be in app.css not in the help, because it refers to element outside the ga-help component.
Please use more sccs syntax : 

```
.cadastre.de, .cadastre.fr {
   #catalogHeading [ga-help] {
       display: none;
   }
}
```